onnxruntime/contrib_ops/cpu/bert/multihead_attention.cc

```diff
@@ -293,20 +293,6 @@ Status MultiHeadAttention<T>::Compute(OpKernelContext* context) const {
 
   AllocatorPtr allocator;
 
-  if (qkv_bias == nullptr) {
```

Comment:
why remove these code?

Reply:
These part causes tests failure, because QKV is not correctly transposed before sending into `ApplyAttention`